.github/workflows/CI.yml

```diff
@@ -0,0 +1,95 @@
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events
+  push:
+  pull_request:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    strategy:
+      matrix:
+        include:
+          - name: "MoltenVK (macOS)"
+            platform: "macos"
+            scheme: "MoltenVK Package (macOS only)"
+            demos-scheme: "Cube-macOS"
+          - name: "MoltenVK (iOS)"
+            platform: "ios"
+            scheme: "MoltenVK Package (iOS only)"
+            demos-scheme: ""
+          - name: "MoltenVK (tvOS)"
+            platform: "tvos"
+            scheme: "MoltenVK Package (tvOS only)"
+            demos-scheme: ""
```

Comment:
I'd suggest adding one for Catalyst, too, but there doesn't seem to be a `(Mac Catalyst only)` scheme. Someone just hit a build error (#1176) on Catalyst, and we'd like to avoid that in the future.

Reply:
>there doesn't seem to be a (Mac Catalyst only) scheme

Perhaps CI can just run `make`? With the right args, it can build for whichever platforms we want, including Mac Cat.

Reply:
I’ll revise how the workflow works, so we can get additional platforms easily. 👍

Reply:
The full `fetchDependencies` build for MoltenVK can take up to half an hour at times. As a result, the Travis CI build was just set up as a smoke test running on macOS only.

We can explore moving to a full CI on all platforms if that is necessary, desirable, and functional.

Reply:
Resolved - Mac Catalyst platform added.